### DISCUSSION TITLE
CPU and system temp

### DISCUSSION THREAD

**Discussion by phdevpro** [alfiosalanitri/server-guardian#5] [General]
Hi,
the script is amazing.
Do you think is feasible to add also the system and CPU temperatures with the same param logic ?
Thanks.

**Comment by alfiosalanitri** [comment]
> Hi, the script is amazing. Do you think is feasible to add also the system and CPU temperatures with the same param logic ? Thanks.

Hi, thanks. 
It's difficult to find a generic working way to get this info. Not everybody (me too) has physical server and virtual cloud servers haven't temperature problem. 
Do you have a raspberry with rasbian? I have a python script that send me an email when the temperature is too high and shut down the pi automatically.

**Comment by phdevpro** [comment]
Hi,
thank you for the reply.
I have a home made server and I would like to monitor the temperature as
well.
Thanks.

Il Mer 18 Mag 2022, 21:45 alfiosalanitri ***@***.***> ha
scritto:

**Reply by alfiosalanitri** [reply]
Can you open your terminal, digit `sensors` and paste the results here?
